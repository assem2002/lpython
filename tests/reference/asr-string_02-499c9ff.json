{
    "basename": "asr-string_02-499c9ff",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/string_02.py",
    "infile_hash": "ed6511565e893791a4bd8ea0b4750817bab13cd6dc0731332127bf58",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-string_02-499c9ff.stderr",
    "stderr_hash": "a00a7d55c016fde4cac60339fd9e8ea14d7b2049ab26366b699dfee7",
    "returncode": 2
}