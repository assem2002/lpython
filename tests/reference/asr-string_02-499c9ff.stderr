semantic error: str.join() takes type list only
 --> tests/errors/string_02.py:6:15
  |
6 |     res:str = x.join(p)
  |               ^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
